docs/source/additional/release.rst

```diff
@@ -8,13 +8,13 @@ Compatibility table
 
 These sets of versions have been tested for compatibility:
 
-.. compatibilitytable:: releases.yaml
+.. compatibilitytable:: releases.json
 
 
 .. only:: html
 
    .. note::
-      `JSON version of the release table </releases.json>`_
+      `JSON version of the release table <../releases.json>`_
```

Comment:
I think the `../` won't work, the path should remain the same with the `html_extra_path` 🤔

Reply:
As we're not able to explain what's exactly happening between dev and prod builds ; I would favor trying this.

---

The path in `html_extra_path` is related to the source file. The `html_extra_path` puts the `releases.json` file at the root of the `build/html` directory:

```txt
build/html
...
├── additional
│   ├── community.html
│   ├── faq.html
│   ├── glossary.html
│   ├── privacy-strategy.html
│   └── release.html
...
├── releases.json
...
```

It's working both in local mode and [dev build](https://owkin-substra-documentation--366.com.readthedocs.build/en/366/releases.json) so at worst it won't break anything.

WDYT?

Reply:
Ooooooh I get it, that's clever! It will break if we move the page (maybe add a comment somewhere) but I think that's our best bet.

Reply:
It will indeed. Not sure how to prevent this ATM though!